styleguide/derek/_partials/solutions/header-smallIMAC.ejs

```diff
@@ -0,0 +1,13 @@
+<!--Header Short-->
+<div class="imacBanner small">
+  <div class="imacBanner-container">
+    <div class="imacBanner-textRow">
+      <div class="imacBanner-textContainer imacBanner-headlineSmall">
+        <h2 class="imacBanner-headline">the cloud runs on its own,<br/>with lots of help</h2>
```

Comment:
Should these have ending punctuation and start with capital letters? 

Yes I know the text-transform does that already, but It just looks cleaner if the grammar is proper. I think if people are using this as their point of reference for our branding styles, it shouldn't create the impression that grammar is unimportant, but that's just my two cents.